test/support/setupTestFramework.js

```diff
@@ -1,3 +1,25 @@
+require('babel-register')({
+  presets: ['es2015', 'es2016', 'react-native'],
+  ignore: /node_modules\/(?!react-native)/,
+})
+require('react-native-mock/mock')
```

Comment:
Requiring this module has the side-effect of overwriting the `require` function's cache to return a mocked-out version of react-native whenever it's imported.